3-JSS/utils/localstorage.js

```diff
@@ -0,0 +1,26 @@
+export class LocalStorage {
+  constructor() {
+    if (!localStorage.getItem("todoList"))
```

Comment:
it's a bit weird to have such side effect on the `constructor` and not on-demand. for example when you're attempting to save todos for the first time

Reply:
Alright, I understand. So now I am checking if the list exists only on save function